release-notes-1.2.0.md

```diff
@@ -0,0 +1,766 @@
+# Overview of jBallerina 1.2.0
+
+JBallerina 1.2.0 is the first major Q1 release of 2020 and it includes a new set of features and significant improvements to the compiler, runtime, standard library and developer tooling. It is based on Stable Language Spec version 2020R1.
+
+# Highlights
+
+- Redesigned XML, Stream and Query support
+- Based on a stable language specification: 2020R1
+- Improved Compilation time.
+- Includes a number of important bug fixes
+- Added cookies and trailer support in HTTP.
+- Revamped Cache module.
+- Redesigned oneof field and map field support in gRPC and Added client retry support for gRPC blocking unary calls.
+- Added custom serializer/deserializer and Avro support in the Kafka connector. 
+- Deprecated previous JDBC connector and introduced new Database connectors which are now in a preview state.
+- Introduced new client connectors (email, slack and azure time-series).
+- Changed organization names of all supported connectors.
+- Support to mock functions when writing tests
+- Test and code coverage report
+- Java binding generator tool
+
+# What's new in JBallerina 1.2.0?
+
+## Language
+
+The language implementation is based on the stable language specification version 2020R1. This implementation introduces a new set of features aimed at making integration further easier for the user. In addition to that, the XML and Lock implementations are out in their preview/experimental status.
+
+In addition to new features, this release also includes critical bug fixes and fixes done to align the implementation with the language specification. Some of these changes are backward incompatible. For a complete list of changes done since 1.1.0, see the `Language changes since JBallerina 1.1.0` section.
+
+### New XML design
+
+This release introduces a revamped XML support along with XPath-like query syntax allowing easy and safe manipulation of XML data. 
+
+#### XML navigation expression
+
+XML step expressions allow to query the children of an XML element or children of members of an XML sequence.
+
+```ballerina
+xml x = xml `<root>
+<person><name><fname>Kamal</fname><lname>Gunae</lname></name></person>
+<person><name><fname>Nimal</fname><lname>Jayee</lname></name></person>
+<person><name><fname>Sunil</fname><lname>Kumarae</lname></name></person>
```

Comment:
@hasithaa had a comment on the doc regarding the names used here. @rdhananjaya

Reply:
Also, the formatting seems off isn't it? Of the `xml` I mean

Reply:
@pubudu - Can you fix the formatting here itself so that I can commit directly?